testbed/tests/test_factory.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+import toga
+
+
+def test_importing_not_existent_gtk_submodule():
+    with pytest.raises(NotImplementedError):
+        toga.platform.get_platform_factory().notamodule
```

Comment:
This isn't a GTK test - it's a test for any platform:
```suggestion
def test_missing_widget():
    with pytest.raises(NotImplementedError):
        toga.platform.get_platform_factory().MissingWidget
```